plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1539,6 +1539,22 @@ public void testTableWithTrailingSlashLocation(boolean partitioned)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAnonymousRowTypeFailure()
+    {
+        String expectedMessage = "Anonymous row type is not supported in Delta Lake connector";
+        String tableName = "test_anonymous_row_type_" + randomNameSuffix();
+
+        String createTable = "CREATE TABLE " + tableName + " AS SELECT ROW(123) AS c1";
```

Comment:
Please also add CREATE TABLE statement because the internal path is different from CREATE TABLE AS SELECT statement.